### PR TITLE
management cluster for MVP

### PR DESCRIPTION
**Pull request by geoberle**
### What this PR does

* introduce the management cluster for MVP
* introduce dedicated bicepparam configs for MVP clusters
* move MVP infra to westus3 to mitigate capacity problems in eastus

Jira: https://issues.redhat.com/browse/ARO-8568
Link to demo recording:

### CODE REVIEW

**Review by tonytheleg — DISMISSED** [review]

**Review by tonytheleg — APPROVED** [review]

File: dev-infrastructure/configurations/mvp-mgmt-cluster.bicepparam [hunk 1]
```diff
@@ -0,0 +1,27 @@
+using '../templates/mgmt-cluster.bicep'
+
+param kubernetesVersion = '1.29.2'
+param istioVersion = 'asm-1-20'
+param vnetAddressPrefix = '10.132.0.0/14'
+param subnetPrefix = '10.132.8.0/21'
+param podSubnetPrefix = '10.132.64.0/18'
+param enablePrivateCluster = false
```
Comment (bennerv) [inline dev-infrastructure/configurations/mvp-mgmt-cluster.bicepparam:8]: do we need private cluster anymore?

File: dev-infrastructure/configurations/mvp-svc-cluster.bicepparam [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+param kubernetesVersion = '1.29.2'
+param istioVersion = 'asm-1-20'
+param vnetAddressPrefix = '10.128.0.0/14'
+param subnetPrefix = '10.128.8.0/21'
+param podSubnetPrefix = '10.128.64.0/18'
+param enablePrivateCluster = false
+param persist = true
+param aksKeyVaultName = 'aks-kv-aro-hcp-dev-sc'
+param disableLocalAuth = false
```
Comment (bennerv) [inline dev-infrastructure/configurations/mvp-svc-cluster.bicepparam:11]: what's the plan to switch this?  

also what's the `disableLocalAuth` apply to?  Cosmosdb, postgres, etc.  Do we need to split them out?

File: dev-infrastructure/configurations/mvp-svc-cluster.bicepparam [hunk 3]
```diff
@@ -0,0 +1,41 @@
+using '../templates/svc-cluster.bicep'
+
+param kubernetesVersion = '1.29.2'
+param istioVersion = 'asm-1-20'
+param vnetAddressPrefix = '10.128.0.0/14'
+param subnetPrefix = '10.128.8.0/21'
+param podSubnetPrefix = '10.128.64.0/18'
+param enablePrivateCluster = false
```
Comment (bennerv) [inline dev-infrastructure/configurations/mvp-svc-cluster.bicepparam:8]: Same as above.  Do we just make these public now?

**Review by bennerv — APPROVED** [review]
lgtm, few questions

File: dev-infrastructure/configurations/mvp-svc-cluster.bicepparam (hunk 2 above)
Comment (geoberle) [reply dev-infrastructure/configurations/mvp-svc-cluster.bicepparam:11]: applies to cosmos only / the postgres module is configured with entra auth only

File: dev-infrastructure/configurations/mvp-mgmt-cluster.bicepparam (hunk 1 above)
Comment (geoberle) [reply dev-infrastructure/configurations/mvp-mgmt-cluster.bicepparam:8]: not to my knowledge

Comment (mjlshen) [reply dev-infrastructure/configurations/mvp-mgmt-cluster.bicepparam:8]: 99% no, but we've been holding until https://issues.redhat.com/browse/ARO-7842 completes to remove this

**Pull request merged**